src/effects/effectknobparameterslot.h

```diff
@@ -1,68 +1,62 @@
-#ifndef EFFECTPARAMETERSLOT_H
-#define EFFECTPARAMETERSLOT_H
+#pragma once
 
 #include <QObject>
 #include <QVariant>
 #include <QString>
 
 #include "control/controlobject.h"
-#include "effects/effect.h"
 #include "effects/effectparameterslotbase.h"
 #include "util/class.h"
 
 class ControlObject;
 class ControlPushButton;
 class ControlEffectKnob;
 class SoftTakeover;
+class EffectSlot;
 
-class EffectParameterSlot : public EffectParameterSlotBase {
+/// Refer to EffectParameterSlotBase for documentation
+class EffectKnobParameterSlot : public EffectParameterSlotBase {
     Q_OBJECT
   public:
-    EffectParameterSlot(const QString& group, const unsigned int iParameterSlotNumber);
-    virtual ~EffectParameterSlot();
+    EffectKnobParameterSlot(const QString& group, const unsigned int iParameterSlotNumber);
+    virtual ~EffectKnobParameterSlot();
 
     static QString formatItemPrefix(const unsigned int iParameterSlotNumber) {
         return QString("parameter%1").arg(iParameterSlotNumber + 1);
     }
 
-    // Load the parameter of the given effect into this EffectParameterSlot
-    void loadEffect(EffectPointer pEffect);
+    // Load the parameter of the given effect into this EffectKnobParameterSlot
```

Comment:
```suggestion
    /// Load the parameter of the given effect into this EffectKnobParameterSlot
```

Reply:
This comment doesn't say anything that the function signature doesn't so I just removed it. e82522b03510328016ed5b5d71de654d641b5d61